### ISSUE TITLE
Using the arrow keys in the REPL should cycle through recently used commands

### ISSUE THREAD

**Issue by malte-v** [edwinb/Idris2-boot#297]
Currently using the arrow keys just prints ANSI sequences, which is different from the Idris 1 REPL.

# Steps to Reproduce
Enter the Idris 2 REPL and use the arrow keys.

# Expected Behavior
It cycles through the recently used commands.

# Observed Behavior
Escape sequences get printed.

**Comment by gallais** [comment]
Note that you can use rlwrap to cope with the current limitations

**Comment by edwinb** [comment]
Yes, I run through `rlwrap` (in fact I've aliased `idris2`). This is a fairly big feature request, in practice, requiring readline support or similar, and then leads to wanting tab completion of function names. I actually wonder if a better way is to write a separate fancy REPL via the IDE protocol, and to keep what Idris 2 itself does as simple as possible.
